### PR TITLE
Check if indexed RAM bundle and loadFromString

### PR DESCRIPTION
**Pull request by dratwas**

### CODE REVIEW

File: ReactCommon/cxxreact/Instance.cpp [hunk 1]
```diff
@@ -108,6 +108,24 @@ bool Instance::isIndexedRAMBundle(const char *sourcePath) {
...
+  strncpy(reinterpret_cast<char *>(&header), script->get()->c_str(), sizeof(header));
+
+  return parseTypeFromHeader(header) == ScriptTag::RAMBundle;
+}
+
+void Instance::loadRAMBundleFromString(std::unique_ptr<const JSBigString> script, const std::string& sourceURL) {
+  auto bundle = folly::make_unique<JSIndexedRAMBundle>(std::move(script));
+  auto startupScript = bundle->getStartupCode();
+  auto registry = RAMBundleRegistry::singleBundleRegistry(std::move(bundle));
+    loadRAMBundle(
```
Comment (zamotany) [inline ReactCommon/cxxreact/Instance.cpp]: Please fix indentation

Comment (thymikee) [reply ReactCommon/cxxreact/Instance.cpp]: FYI you should be good running `yarn clang-format`

File: ReactAndroid/src/main/jni/react/jni/CatalystInstanceImpl.cpp [hunk 2]
```diff
@@ -197,6 +197,8 @@ void CatalystInstanceImpl::jniLoadScriptFromAssets(
       sourceURL,
       loadSynchronously);
     return;
+  } else if (Instance::isIndexedRAMBundle(&script)) {
+    instance_->loadRAMBundlefromString(std::move(script), sourceURL);
```
Comment (zamotany) [inline ReactAndroid/src/main/jni/react/jni/CatalystInstanceImpl.cpp]:
```suggestion
    instance_->loadRAMBundleFromString(std::move(script), sourceURL);
```

**Comment by zamotany** [comment]
Closing this, since we have a PR open in React Native repo itself.

**Pull request closed without merging**